src/07-projects/project-items.apib

```diff
@@ -120,4 +120,20 @@ Get details for a single project item.
                 + type: `product` (string)
                 + id: `e2314517-3cab-4aa9-8471-450e73449040` (string)
             + added_at: `2018-12-04T16:44:33+00:00` (string)
-            + updated_at: `2018-12-05T16:44:33+00:00` (string)
\ No newline at end of file
+            + updated_at: `2018-12-05T16:44:33+00:00` (string)
+
+
+### projectItems.setParticipantRates [POST /projectItems.setParticipantRates]
+
+Set participant user rates for timetracking project items, optionally for a single project
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + project_id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, optional)
+        + participant_id: `b5094b3f-bb7a-0391-b01b-e709773f3509` (string)
+        + internal_cost (Money, nullable)
```

Comment:
PP: I would use `internal_hourly_rate`, since we use the word "rate" in the endpoint.